### PR TITLE
Cleared TODO comments across all files - task #171

### PR DESCRIPTION
**Pull request by khleomix**
For task #171 

Cleared all TODO comments and added header summaries for those that need it. Also, took out header summary period on files that have them.

### CODE REVIEW

**Comment by kjellr** [comment]
Thank you for tackling this, @khleomix! I'm seeing a handful of [minor Travis errors](https://travis-ci.org/WordPress/twentynineteen/jobs/443728711). Once those are sorted out, this should be good to go.

**Comment by khleomix** [comment]
@kjellr saw the errors, working on that now. two files from the classes folder are reporting "Missing file doc comment" though

**Comment by kjellr** [comment]
Looks great! Thanks for the PR, @khleomix!

File: footer.php [hunk 1]
```diff
@@ -18,7 +18,6 @@
 		<div class="site-info">
 			<a class="site-name" href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php bloginfo( 'name' ); ?></a>,
 			<a href="<?php echo esc_url( __( 'https://wordpress.org/', 'twentynineteen' ) ); ?>" class="imprint">
-				<?php /* translators: TODO comment on placeholders */ ?>
```
Comment (grappler) [inline footer.php]: This should not have been deleted but a proper comment should have been added.

**Comment by khleomix** [comment]
@kjellr please see: #237 for the fix. Thanks

**Pull request merged**